components/sheet/ExportDataSheet.tsx

```
@@ -0,0 +1,261 @@
+  Database,
+  Eye,
+} from '@tamagui/lucide-icons';
+import { backgroundStyle, handleIndicatorStyle, snapPoints } from './constants';
+import { DatePicker } from '../DatePicker';
+import useTransactionExport, {
+  ExportColumn,
+  ExportDateRange,
+  ExportDestination,
+} from '../../hooks/useTransactionExport';
+import dayjs from 'dayjs';
+import { Platform } from 'react-native';
```

ESLint

[error] 'Platform' is defined but never used.

(@typescript-eslint/no-unused-vars)